Service/src/routes/stock.py

```diff
@@ -13,127 +13,136 @@
 MONTH = '1m'
 YEAR = '1y'
 
-IEX_DATETIME_FORMAT = '%Y%m%d %H:%M'
+IEX_DATETIME_FORMAT = '%Y-%m-%d %H:%M'
 IEX_DATE_FORMAT = '%Y-%m-%d'
 DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 
 stock_api = Blueprint('stock_api', __name__)
 
-IEX_URL_PREFIX = 'https://api.iextrading.com/1.0/stock/'
+IEX_URL_PREFIX = 'https://cloud.iexapis.com/v1/stock/'
+
+CONFIG_FILE_PATH = './config.json'
+
+# This needs to be optimized so it doesn't open and close the config file every time.
+def getIexToken():
+	try:
+		configFile = open(CONFIG_FILE_PATH, 'r')
+		config = json.load(configFile)
+		configFile.close()
+		return config['iexToken']
+	except Exception as e:
+		raise Exception('Could not retrieve the iexToken') 
 
 @stock_api.route('/api/v1.0/stock/<ticker>', methods=['GET'])
 @auth.login_required
 def getStock(ticker):
-	stockInformation = getStockInformation(ticker)
-
-	if (stockInformation == None):
-		return make_response(jsonify(InvalidTicker=errors['unsupportedTicker']), 400)
+	try:
+		stockInformation = getStockInformation(ticker)
+	except requests.HTTPError as e:
+		return handleIexError(e)
 
 	return json.dumps(stockInformation)
 
 def getStockInformation(ticker):
-	requestUrl = IEX_URL_PREFIX + ticker + '/company'
+	requestUrl = f'{IEX_URL_PREFIX}{ticker}/company?token={getIexToken()}'
```

Comment:
this is kinda breaking pattern since we do + concat everywhere. It's fine with me if we don't have one way of doing it but just wanted to point it out

Reply:
If we want to follow a standard, I think f strings are the way to go. It's a lot cleaner than concatenating string. I'll make a task for updating other string since converting all of them would be out of scope for this task.